README.md

```diff
@@ -96,7 +96,8 @@ accomplish this as follows:
 
 2.  Similarly, user agents would freeze the `navigator.appVersion`, `navigator.platform`,
     `navigator.productSub`, `navigator.vendor`, and `navigator.userAgent` attributes to
-    appropriate values for the frozen `User-Agent` string.
+    appropriate values for the frozen `User-Agent` string, along with a `navigator.mobile` attribute signifiying if
```

Comment:
(independent of whether this is a good idea or not, it seems like an extension to `NavigatorID` should happen in HTML: https://html.spec.whatwg.org/multipage/system-state.html#navigatorid)

Reply:
That was actually a mis-copy. I'll go ahead and remove it.